centos_ci.sh

```diff
@@ -305,7 +305,7 @@ function build_and_test() {
   cd $GOPATH/src/github.com/minishift/minishift
   make prerelease synopsis_docs link_check_docs
   # Run integration test with 'kvm' driver
-  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags=basic,openshift,version,cmd-config -format pretty"
+  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags basic,cmd-addons -format pretty"
```

Comment:
@agajdosi I think you should remove this temporary change and update the PR as it will allow it run the complete tests before we merge it.

Reply:
@LalatenduMohanty ack